.github/workflows/test.yaml

```diff
@@ -9,19 +9,19 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
           python-version: 3.9
-      - uses: pre-commit/action@v3.0.0
+      - uses: pre-commit/action@v3.0.1
 
   pytest:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
```

Comment:
are you missing the step `- run: pipenv run coverage report -m` to create coverage reports before `- name: upload coverage to coveralls`?

Reply:
no, I actually meant to remove that line from the other PR, I was just using it for debugging. the test is failing because of the upstream issue [linked in the other PR](https://github.com/Sage-Bionetworks-IT/lambda-ec2-terminator/pull/1#issuecomment-2048153791) and I chose to leave this PR blocked on the issue instead of port the workaround here